thread-inactive.go

```diff
@@ -6,15 +6,13 @@ import (
 
```

Comment:
```suggestion
threadinactive.go
```

IIRC it's not common in go to use dashes in filenames.